### ISSUE TITLE
[Action Group] a11y review

### ISSUE THREAD

**Issue by Westbrook** [adobe/spectrum-web-components#1419]
### Feedback

- It should be possible to label the group/radiogroup using aria-label or aria-labelledby. Anything expressed in aria-label is currently not reflected in the DOM. We should provide an optional attribute to name the actionGroup

### Path forward

1. ~We currently offer a `label` attribute for this. Should a labelled Action Group be preferred via additional documentation regarding accessible content here? Also, are you saying that you attempted to apply `aria-label` and it was somehow removed? If so, could you share some repro steps on that?~
2. Expand docs to outline the usage of `label` attribute.

cc @jnurthen

**Comment by Westbrook** [comment]
> Yes - see
> https://webcomponents.dev/edit/k06pwox0ezm0WJUGK6bC/

Ah, I see. I'm not sure how I wasn't able to get this. We're resetting on initialization, that can be focused in a way that is more expected.
